README.md

```diff
@@ -1 +1,13 @@
 # hw01
+
+##*Ta'er*, Seer of Paths, Traverser of Seas
+All hail the great *Ta'er*, Lord of the Storm! Raised as Seth Markow in the humble coastal city of *Saun Dieagoh*, *Ta'er* began preaching his gospel in the tundra city of *Shikagho*, and soon his teachings became law, and spread across the world, ushering in a new era of peace and prosperity for the people of Earth. Now, as we venture further and further into the outer reaches of space, soon others, too, will know the goodness of *Ta'er*, Knower of the Way, Seer of the Future!
+ 
+I've been reading Dune lately and it's definitely influenced me.
+
+#Seth
+But actually. I am indeed from San Diego, which I return to periodically to surf with my mom, run by myself, and drink cocktails with my dad. My first two winters in Chicago were fun and romantic out of novelty but now I'm over it and plan to live out the rest of my life in a place where it almost never gets above 85 degrees and truly never drops below 50.
+Here in Chicago, I used to do a lot of things. But ever since the pandemic hit I mostly spend my time biking around, baking, and staring at walls. Check out this pie I made a few months ago: ![imageofpie](https://lh3.googleusercontent.com/XBzhofGtyE9YTVvE4t4edP4fMeKQu8s7eWgYWdBE4PLDb_F0Zk0Bg4GIkxnmTCBgQqkylLoqN_JLLGEu4WnU3Px5o3P_zvmmVesp-MCgOcWpCanPD0i2sOuOHXj2YwYm34F9HVuw3ju4x-geQMKigBAZf_fHPLUCGmkr6m2QpB24ALfc3T46xNbPecQJC4bKiMkFxW8J9PN3-3ueAM2tXCsSpXE8Js94C2hzJst5KovSAFOm2j0K_JHXB6c9p6QV3B95aJENVhzP4tvZIe39ZUNhozvjd0HaRZyE86OrlLjNd7vdXxRpZLuSGfz35Obeo07l5ksrHpux8j2IEjtnXjU-SvSAiK2tehnpbuLXc4nO5cWR8laDKMpCTBHWHY1dvg4JhoStrD7epLRY9bDbBBD1qmm7tHAAO4KafLlkEwryk1nVeP6saBqE-FSvyerhQxSC4VEaivw9UEmfxBDTvlu0xlzcWgdfwzNkQf4f2dDSsL123u4a9BUgrle2wciCP5kcFwawu95A2eC9Twco8zlkOrtXSGTzUu9huSmwP4k_GexM3yLE-tNUQzei7jv0NWsvpm2aojVCkwP21aNsph8gHUlXkOhtu-gaH6qsQcc6zkjLudF0Im7N_PEcWK_mM9dRk0jLDF5_2VuBgk4JyCNLY97tiq8FxwmMEFyXRxk8e_gKkxgwlA29=w632-h842-no?authuser=0)
+I'm a Sociology and Near Eastern Languages and Civilization Double Major, which is probably only the third least employable combination of fields of studies I've seen at this school.
+>Well, that's all folks!
+>-Looney Tunes Signout
```

Comment:
I love your signoff! It's a great way to end the bibliography and choosing to put it in a different color than the rest of your text was really clever!